recipes/source-highlight/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "source-highlight" %}
+{% set version = "3.1.8" %}
+{% set sha256 = "7cc44a905410d63b23f77877b3ceec98f07bedc086b2137880904ac8c1ad4877" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://git.savannah.gnu.org/cgit/src-highlite.git/snapshot/src-highlite-rel_{{ version }}.tar.gz
```

Comment:
This works, but probably better to use the official release tarball:
```
https://ftp.gnu.org/gnu/src-highlite/source-highlight-{{ version }}.tar.gz
```
which has sha256
`01336a7ea1d1ccc374201f7b81ffa94d0aecb33afc7d6903ebf9fbf33a55ada3`.

Reply:
thanks, will do